app/javascript/crayons/MentionAutocomplete/__tests__/MentionAutocomplete.test.jsx

```diff
@@ -0,0 +1,29 @@
+import { h, Fragment, createRef } from 'preact';
+import { render } from '@testing-library/preact';
+import '@testing-library/jest-dom';
+import { axe } from 'jest-axe';
+import { MentionAutocomplete } from '../MentionAutocomplete';
+
+describe('<MentionAutocomplete />', () => {
```

Comment:
The majority of MentionAutocomplete needs tested with Cypress, which will be done in the next PR once we implement this in the app instead of just storybook